template.go

```diff
@@ -166,7 +166,7 @@ func overrideParam(s *string, newS string) {
 	}
 }
 
-func UnmarshalTemplate(bytes []byte) (*template, error) {
```

Comment:
since this is a `main.go` and can't be imported, should be OK since it won't break anything.